blog/2025/devnet-reset.md

```diff
@@ -0,0 +1,68 @@
+---
+category: 2025
+date: 2025-01-24
+seo:
+    description: Devnet is scheduled to reset on Monday, February 3, 2025 due to Clio databases nearing capacity. Learn more.
+labels:
+    - Advisories
+markdown:
+    editPage:
+        hide: true
+---
+# Upcoming Devnet Reset
+
+Devnet is scheduled for a reset on **Monday, February 3, 2025**. Clio server databases on the network are nearing storage capacity and need to be cleared.
+
+<!--
+{% admonition type="success" name="Update" %}
+The reset has completed successfully. Devnet is online and fully operational.
+{% /admonition %}
+-->
+
+## Background
+
+Ripple operates Clio servers on Devnet to help the community access validated ledger data efficiently. Clio handles API calls better and stores data in a more space-efficient format than `rippled` servers, ultimately reducing the load on the network.
+
+The virtual machines we're running the Clio servers from are nearing storage capacity. In order to continue serving the community, Devnet will be reset to clear the Clio databases.
```

Comment:
Without a byline, the "we" here is ambiguous. This is the XRPL dev blog, not a Ripple blog, so it shouldn't be assumed that "we" are "Ripple" in this context. It might be better to avoid 1st person entirely.

```suggestion
The virtual machines running the Clio servers are nearing storage capacity. In order to continue serving the community, Devnet will be reset to clear the Clio databases.
```